sirepo/template/srw.py

```diff
@@ -1498,7 +1525,7 @@ def _generate_srw_main(data, plot_reports):
     run_all = report == _RUN_ALL_MODEL
     content = [
         'v = srwl_bl.srwl_uti_parse_options(varParam, use_sys_argv={})'.format(plot_reports),
-        'source_type, mag = srwl_bl.setup_source(v)',
+        #'source_type, mag = srwl_bl.setup_source(v)',
```

Comment:
Why is it commented out?

Reply:
@mrakitin, I think this and the ```mag``` parameter change below have been integrated into SRW